### PR TITLE
Release v0.24.0

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nakabonne** [comment]
Thank you for preparing the notes!
/lgtm

File: docs/content/en/blog/releases/v0.24.0.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+* Filtering applications by Labels is now available ([#2987](https://github.com/pipe-cd/pipe/pull/2987))
+
+### Deprecated Features
+
+Environment concept has been deprecated. Please use Label instead.
+
+* Disable the ability to add environment from control-plane ([#3049](https://github.com/pipe-cd/pipe/pull/3049))
+* Environment field has been removed from application registration form ([#3049](https://github.com/pipe-cd/pipe/pull/3051))
+
+Deployment configuration file has been deprecated. Please use Application configuration file instead, it has the same layout but with some new field such as `name`, `description` and `labels`. By using this new config file, all application information can be stored and managed only in your Git repository. Basically, a deployment configuration file can be migrated to be an application configuration file just by adding the missing fields, but you can also use this prepared `pipectl` command to migrate a bunch of your files.
```
Comment (khanhtc1202) [inline docs/content/en/blog/releases/v0.24.0.md:26]:
```suggestion
The deployment configuration file has been deprecated. Please use the Application configuration file instead, it has the same format but with some new fields such as `name`, `description`, and `labels`. By using this new config file, all application information can be stored and managed only in your Git repository. Basically, a deployment configuration file can be migrated to be an application configuration file just by adding the missing fields, but you can also use this prepared `pipectl` command to migrate a bunch of your files.
```

Comment (nghialv) [reply docs/content/en/blog/releases/v0.24.0.md]: Applied!

**Comment by khanhtc1202** [comment]
🚀 🚀 🚀 
/approve

**Pull request merged**